.env.example

```diff
@@ -17,6 +17,20 @@ POSTGRES_URL=postgresql://postgres:password@localhost:5432/conference?schema=pub
 NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
 CLERK_SECRET_KEY=
 
-# Regarding the two Flowcore environmental variables, it is the datacore url (in the format https://webhook.api.flowcore.io/event/orgname/datacoreuuid, where orgname is your organization's slug and datacoreuuid is the uuid of your datacore) and the organization' api key (found at the bottom of e.g. https://flowcore.io/orgname/settings, where orgname is your organization's slug).
-FLOWCORE_DATACORE=
-FLOWCORE_KEY=
\ No newline at end of file
+# Flowcore base url (you rarely need to change this one)
+FLOWCORE_WEBHOOK_BASEURL = "https://webhook.api.flowcore.io"
+
+# Flowcore webhook base url (you rarely need to change this one)
+FLOWCORE_BASEURL = "https://flowcore.io"
+
+# Flowcore tenant (the organization name in Flowcore)
+FLOWCORE_TENANT = ""
+
+# Flowcore datacore name
+FLOWCORE_DATACORE = ""
+
+# Flowcore api key (used by webhook)
+FLOWCORE_KEY = ""
+
+# Flowcore transformer secret
```

Comment:
could write a little segment within parenthesis, that the secret name can be anything, but must match here and on the proxy adapter? 🤔